src/Compute/Compute/Generated/Models/PSProximityPlacementGroup.cs

```diff
@@ -41,9 +41,10 @@ public string ResourceGroupName
         }
 
         public string ProximityPlacementGroupType { get; set; }
-        public IList<SubResource> VirtualMachines { get; set; }
-        public IList<SubResource> VirtualMachineScaleSets { get; set; }
-        public IList<SubResource> AvailabilitySets { get; set; }
+        public IList<SubResourceWithColocationStatus> VirtualMachines { get; set; }
```

Comment:
To change a return property's type is a breaking change, please follow the breaking change release process.

- add a breaking change notification to the current GA module (refer to https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md#output-and-parameter-types)
- create a preview branch to contain your breaking change (optional) 
- breaking change goes GA in the next May.